crates/proto/src/rr/dns_class.rs

```diff
@@ -48,6 +48,7 @@ impl FromStr for DNSClass {
     /// assert_eq!(DNSClass::IN, var);
     /// ```
     fn from_str(str: &str) -> ProtoResult<Self> {
+        debug_assert!(str.chars().all(|x| char::is_ascii_uppercase(&x)));
```

Comment:
Here, the `char::is_ascii_uppercase()` function worked great! Thanks for the review.